src/Console/Application.php

```diff
@@ -147,10 +157,34 @@ private function addCustomOptions(InputDefinition $inputDefinition): void
             InputOption::VALUE_NONE,
             'Enable debug verbosity (-vvv)'
         ));
+
+        $inputDefinition->addOption(new InputOption(
+            '--working-dir',
+            '-d',
+            InputOption::VALUE_REQUIRED,
+            'If specified, use the given directory as working directory.'
+        ));
```

Comment:
I talk only about this section

Reply:
I can't get them displayed w/ --help when I put them in the abstract command. I assume this is b/c I want to set a global option which is an option of the *application* and a *command* only can configure non-global options.

Reply:
Ah, that's problem of this coupling. It should be separated service.

Thanks for investigation. This is good enough